api_v3/services/ServerNodeService.php

```diff
@@ -188,4 +190,26 @@ private function addNewServerNode(KalturaServerNode $serverNode)
 		
 		return $dbServerNode;
 	}
+
+	/**
+	 * Unregister server node by id
+	 *
+	 * @action unregister
+	 * @param string $serverNodeId
+	 * @throws KalturaErrors::INVALID_OBJECT_ID
+	 * @return KalturaServerNode
+	 * @throws KalturaAPIException
+	 */
+	function unregisterAction($serverNodeId)
+	{
+		$dbServerNode = ServerNodePeer::retrieveByPK($serverNodeId);
```

Comment:
You should probably use retrieveActiveServerNode to not fetch serverNodes which are disabled.